index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <main class="container">
+        <header class="notification">
+            <div class="logo"><img src="img/My Logo.png" alt="Logo"></div>
+            <div class="hamburger"><nav><img src="img/Union.png" alt="menu"></nav></div>
+        </header>
+        <div class="main_body">
+            <div class="header2">I'm Ali<br> Glad to meet you!</div>
+            <div class="body2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</div>
+            <div class="connect">Let's Connect</div>
+            <div class="icons">
+                <div class="twitter"><img src="img/Icon - Twitter.png" alt="Twitter"></div>
+                <div class="linkedin"><img src="img/Icon - Linkedin.png" alt="Linkedin"></div>
+                <div class="medium"><img src="img/Icon - Medium.png" alt="Medium"></div>
+                <div class="github"><img src="img/Icon -GitHub.png" alt="GitHub"></div>
+                <div class="angelist"><img src="img/Icon - Angelist.png" alt="Angelist"></div>
+            </div>
```

Comment:
Kindly update your list of social links to be wrapped in a `<ul>` tag instead of a `<div>` tag. This is for semantic reasons as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`